src/cmd/prune_branches.go

```diff
@@ -38,6 +41,12 @@ This usually means the branch was shipped or killed on another machine.`,
 }
 
 func checkPruneBranchesPreconditions() {
+	if git.IsOffline() {
+		fmt.Println(`Error: cannot prune branches in offline mode
+
+Pruning branches requires an active internet connection.`)
```

Comment:
This is a more in depth error message than #982. We should standardize what we want the offline error message to include. Maybe we can have a function `util.ExitWithOfflineError` where you pass in the a parameter of what command is being run